legal-api/src/legal_api/models/alternate_name.py

```diff
@@ -111,6 +119,40 @@ def find_by_name(cls, name: str = None):
         alternate_name = cls.query.filter_by(name=name).filter_by(end_date=None).one_or_none()
         return alternate_name
 
+    @property
+    def office_mailing_address(self):
+        """Return the mailing address."""
+        if (
+            business_office := db.session.query(Office)  # SP/GP
+            .filter(Office.alternate_name_id == self.id)
+            .filter(Office.office_type == "businessOffice")
+            .one_or_none()
+        ):
+            return business_office.addresses.filter(Address.address_type == "mailing")
+
+        return (
+            db.session.query(Address)
+            .filter(Address.legal_entity_id == self.id)
```

Comment:
`Address.legal_entity_id` should be `Address.alternate_name_id`

Reply:
Thank you for catching that!